webpack.renderer.config.js

```diff
@@ -96,11 +100,15 @@ let rendererConfig = {
   },
   resolve: {
     alias: {
-      'components': path.join(__dirname, 'app/src/renderer/components'),
-      'renderer': path.join(__dirname, 'app/src/renderer'),
-      '@': path.join(__dirname, 'app/src/renderer')
+      'components': resolve('app/src/renderer/components'),
+      'renderer': resolve('app/src/renderer'),
+      '@': resolve('app/src/renderer'),
+      assets: resolve('app/src/renderer/assets'),
+      comp: resolve('app/src/renderer/components'),
+      common: resolve('app/src/renderer/components/common'),
```

Comment:
@faboweb can you help figure out why CircleCI/Jest isn't finding `common/AnchorCopy.vue` when it runs the app unit test? I'm thinking it's related to the Webpack `resolve.alias` module I used:

Reply:
This seems related https://stackoverflow.com/questions/42629925/testing-with-jest-and-webpack-aliases

Reply:
U already found the solution. Jest uses their own mapper because it doesn't touch webpack. Therefor we need to have module mappers in webpack and jest.